docs/dev/security/attestation-internals.rst

```diff
@@ -0,0 +1,348 @@
+Internals and Technical Details for PEP 740 on PyPI
```

Comment:
This file still needs to be moved, it still appears under the "Development" section.

Reply:
Done (I think) -- it looks like I moved it, but I accidentally included an `<h1>` equivalent that put it back at the top level of the `toctree`.